wiki/Internet_Relay_Chat/en.md

```diff
@@ -1,100 +1,84 @@
-| [mIRC](http://www.mirc.com/)              | Tools » Options » pick "IRC". Click the "Events..." button. Change the "joins", "parts", "quits", and "nicks" to your desired settings: "In Status" or "Hide" are good options [1](http://i.clintecker.com/disable-irc-msgs.html).                                       |
-| [Quassel IRC](http://www.quassel-irc.org) | Right click on the chatting window, then choose Hide Events » Join/Part/Quit.                                                                                                                                                                                            |
-| [XChat](http://www.xchat.org)             | Right-click on the tab you want to change. In the submenu of the channelname, there's a toggle-item "Show join/part messages", simply turn this off. Or you type /set irc\_conf\_mode 1 [2](http://xchat.org/faq/#q211) to disable the messages throughout the channels. |
+| IRC Client                                | Description |
+| ----------------------------------------- | ----------- |
+| [HexChat](http://hexchat.github.io/)      | Go to Settings - Preferences, under Chatting - General, check "Hide join and part messages" |
+| [ircII](http://www.eterna.com.au/ircii/)  | Type `/ignore * crap` |
+| [Irssi](http://www.irssi.org)             | Type `/ignore -channels #somechannel * JOINS PARTS QUITS` |
+| [Weechat](http://www.weechat.org)         | Type `/filter add irc_smart_weechat irc.username.#channel irc_smart_filter *` <br> **Note:** Replace **username** to your osu! username.
+| [KVIrc](http://www.kvirc.net)             | Visit [this thread](http://www.kvirc.ru/forum/?topic=609.0) in the official KVIrc forums. |
+| [mIRC](http://www.mirc.com/)              | Go to mIRC Options (Tools - Options / Alt + O), under IRC tree, click "Events..." button and change Joins and Parts to "Hide" |
+| [Quassel IRC](http://www.quassel-irc.org) | Right click on the chatting window, then choose Hide Events » Join/Part/Quit. |
+| [XChat](http://www.xchat.org)             | Type `/set irc_conf_mode 1` (or [2](http://xchat.org/faq/#q211) to disable the messages throughout the channels). |
 
 If your client isn't listed here refer to its documentation, most clients have a way to do this.
 
+
 Frequently Asked Questions (FAQ)
 ================================
-
-I am having an error "Bad Authentication Token". What should I do?
+I am having an error "Bad Authentication Token".
 ------------------------------------------------------------------
-
 1. Make sure you are using the password at the [IRC Authentication](https://osu.ppy.sh/p/irc) page.
+2. If your username had a space, replace it to underscores (e.g. **This Username** to **This_Username**)
```

Comment:
"If your username has spaces, replace them with underscores (e.g. **This Username..."